homeassistant/components/growatt_server/const.py

```diff
@@ -8,13 +8,16 @@
 DEFAULT_NAME = "Growatt"
 
 SERVER_URLS = [
-    "https://server-api.growatt.com/",
-    "https://server-us.growatt.com/",
-    "http://server.smten.com/",
+    "https://openapi-cn.growatt.com/",  # Chinese server
+    "https://openapi-us.growatt.com/",  # North American server
+    "https://openapi.growatt.com/",  # Other regional server
```

Comment:
I suggest to put this one first, since that will be used as the default URL further down.

Reply:
I have moved "openapi.growatt.com" to the top of the list as the default option.